dse-6.8/Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM openjdk:8u252-slim
 
+ARG STARGATE_VERSION=v0.0.5
+ARG PERSISTENCE_VERSION=v0.0.1
 
+RUN apt update && apt install curl unzip iproute2 libaio1 -y
+
+RUN curl -sL https://github.com/stargate/stargate/releases/download/${STARGATE_VERSION}/stargate-jars.zip > stargate-jars.zip
+RUN curl -sL https://github.com/stargate/persistence-extensions/releases/download/${PERSISTENCE_VERSION}/stargate-jars.zip > persistence-jars.zip
+
+RUN unzip -n stargate-jars && unzip -n persistence-jars # Both will unzip directly into the existing stargate-lib directory
+RUN rm stargate-lib/persistence-cassandra-*
```

Comment:
Would it be more robust to unzip DSE jars specifically, as opposed to unzipping all and deleting "cassanda" jars?

Reply:
I wasn't able to find a way to only unzip certain files.

Reply:
The man page for `unzip` indicates it supports wildcards in the list of files to process :thinking:

Reply:
Would something like this work on line 11? 
```
unzip -n persistence-jars 'stargate-lib/persistence-dse*'
```

Reply:
It does. Will get that added. Thanks!